command/git_push_test.go

```diff
@@ -75,6 +87,13 @@ func TestGitPush(t *testing.T) {
 		},
 		"PushPatch": func(*testing.T) {
 			manager := &jasper.MockManager{}
+			manager.Create = func(opts *jasper.CreateOptions) jasper.MockProcess {
+				_, err = opts.Output.Error.Write([]byte(fmt.Sprintf("The key: %s", token)))
```

Comment:
This will end up writing the message twice won't it?

Reply:
I think three times.
I changed the test to just check the last message.